src/View/Rules/index.html.twig

```diff
@@ -0,0 +1,75 @@
+{% extends 'layout.html.twig' %}
+
+{% block stylesheet %}
+	<link rel="stylesheet" href="/assets/css/rules.css">
```

Comment:
you will need the parent function of twig in order to implement global header and footer css files as well